tasks/webfont.js

```diff
@@ -113,11 +114,13 @@ module.exports = function(grunt) {
 		// @todo Codepoint can be a Unicode code or character.
 		var currentCodepoint = o.startCodepoint;
 		if (!o.codepoints) o.codepoints = {};
+		if (o.codepointsFile) o.codepoints = codepointsFromFile();
 		o.glyphs.forEach(function(name) {
 			if (!o.codepoints[name]) {
 				o.codepoints[name] = getNextCodepoint();
 			}
 		});
+		saveCodepointsToFile();
```

Comment:
Added check in read function, the outside check if mainly to prevent codepoints to be overwritten when codepointsFile is not set.

Thanks for your feedback anyway. Helps